server/sockets/paths.js

```diff
@@ -55,10 +55,15 @@ module.exports = (http) => {
     });
 
 
-    socket.on('code-edit', (code) => {
-      socket.to(code.roomName).emit('newCode', code);
-      return utils.sendMessageOrCode(code);
-    });
+    socket.on('code-edit', code => (
+      utils.sendMessageOrCode(code)
+                  .then((codeId) => {
+                    code.mCId = codeId;
```

Comment:
I'm pretty sure there's a less-intensive way to accomplish this same db save, such as socket.on('disconnect', ...). Or perhaps throttle the number of times the listener is fired. We can tackle this later, though.